action.yml

```diff
@@ -29,13 +29,15 @@ runs:
     - name: update status
       shell: bash
       run: |
-        curl -H "Authorization: bearer ${{ inputs.token }}" \
-        -H "Accept: ${{ inputs.headers }}" \
```

Comment:
Good call -- we'd need custom headers if we were accessing an API preview. The `statuses` API, however, is set in stone.